src/twisted/python/test/test_deprecate.py

```diff
@@ -375,6 +375,29 @@ def callTestFunction():
     b = testFunction()
     if b == 3:
         deprecate.warnAboutFunction(testFunction, "A Warning String")
+"""
+        )
+        self.package.child("pep626.py").setContent(
+            b"""
+"A module string"
+
+from twisted.python import deprecate
+
+def noop():
+    pass
+
+def testFunction(a=1, b=1):
+    "A doc string"
+    if a:
+        if b:
+            noop()
+        else:
+            pass
+
+def callTestFunction():
+    b = testFunction()
+    if b is None:
+        deprecate.warnAboutFunction(testFunction, "A Warning String")
```

Comment:
Would this minimal example work?

```suggestion
def callTestFunction():
    deprecate.warnAboutFunction(testFunction, "A Warning String")
    
def otherFunctionNotCalledInTest():
   """
   Here to make sure we don't accidentally just report last line of the file.
   """
   deprecate.warnAboutFunction(testFunction, "Shoulnd't be called")
   more_calls_here()
```

Reply:
No, we need a specific branching structure to trigger the change in dis.findlinestarts output

Reply:
Understood... but in this case I think that a docstring or some comment would help. Thanks!